### PR TITLE
PDF To Word Converter

### PR DESCRIPTION
**Pull request by atharvanaik225**
# Pull Request Template

**PDF to DOCX converter** -

**pdf2docx.py converts the files in 2 ways. The user can either choose to convert using an offline parser, however, certain stylistic elements could be lost. The second option is using a Group Docs cloud conversion tool which will create a better quality word document, however, the user will have to create a free account with group docs and get an App sid and an API key to use this option.**

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in proper folder (Name of folder should be in lowercase with no space in between)
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [x] I have added README.md and requirements.txt with my script

Thank you for the opportunity!

### CODE REVIEW

**Comment by pawangeek** [comment]
@atharvanaik225 fix linting issues

**Comment by atharvanaik225** [comment]
Hey @pawangeek  sorry for the linting errors. So should I now make changes and send another PR?

File: pdf2docx/pdf2docx.py [hunk 1]
```diff
@@ -12,57 +12,47 @@ def convert_pdf2docx(input_file: str, pages: Tuple = None):
...
+
+def cloud_convert(app_sid, app_key):
     convert_api = groupdocs_conversion_cloud.ConvertApi.from_keys(app_sid, app_key)     # Create instance of the API
     file_api = groupdocs_conversion_cloud.FileApi.from_keys(app_sid, app_key)
 
     try:
-
-        filename = input("Please enter the complete and correct file path of the required pdf: ")    # upload source file to storage
+        # upload source file to storage
+        filename = input("Please enter the complete and correct file path of the required pdf: ")
```
Comment (pawangeek) [inline pdf2docx/pdf2docx.py:22]: Remove trailing spaces here

File: pdf2docx/pdf2docx.py [hunk 2]
```diff
@@ -12,57 +12,47 @@ def convert_pdf2docx(input_file: str, pages: Tuple = None):
...
+        convert_pdf2docx(pdf_path)
     else:
-        app_sid = input("Enter app SID: ")
-        app_key = input("Enter api key: ")
-        cloud_convert(app_sid,app_key)
-
+        sid = input("Enter app SID: ")
+        api_key = input("Enter api key: ")
+        cloud_convert(sid, api_key)
+
```
Comment (pawangeek) [inline pdf2docx/pdf2docx.py:58]: Remove space from here

Comment (atharvanaik225) [reply pdf2docx/pdf2docx.py]: Done! sorry for the linting mistakes.

File: pdf2docx/pdf2docx.py (hunk 1 above)
Comment (atharvanaik225) [reply pdf2docx/pdf2docx.py]: Done!

**Comment by pawangeek** [comment]
@all-contributors add @atharvanaik225 for code

**Pull request merged**